tests/drivers/build_all/interrupt_controller/common/boards/imx8mp_evk_mimx8ml8_adsp.overlay

```diff
@@ -22,23 +22,23 @@
 			    compatible = "nxp,irqsteer-master";
 			    reg = <0>;
 			    interrupt-controller;
```

Comment:
Please provide more context in your commit messages. One year later a poor guy who tries to understand this patch wont find any useful info.